doc/changes/changes_0.2.0.md

```diff
@@ -1,7 +1,23 @@
-# bucketfs-client 0.1.1, released 2021-08-31
+# bucketfs-client 0.2.0, released 2021-08-31
 
 Code name: Documentation improvements
 
+## Summary
+
+BucketFS client 0.2.0 contains a *breaking change* in the Bucket URL scheme. We removed the BucketFS service name since it was redundant with the port number. While the service name is easier too remember than a port number, this also led to confusion and potential conflict between port number and service name.
```

Comment:
```suggestion
BucketFS client 0.2.0 contains a *breaking change* in the Bucket URL scheme. We removed the BucketFS service name since it was redundant with the port number. While the service name is easier to remember than a port number, this also led to confusion and potential conflict between port number and service name.
```